spyder/widgets/findinfiles.py

```diff
@@ -13,102 +13,43 @@
 
 # Standard library imports
 from __future__ import with_statement
+from __future__ import unicode_literals

```

Comment:
Also, you can simplify importing from `__future__` like this

    from __future__ import with_statement, print_function